float64 latitude
float64 longitude
float64 altitude
uint16 heading
uint16 speed
uint8 extent
